### ISSUE TITLE
[Bug]: netdata install failing on k8s

### ISSUE THREAD

**Issue by marliyev** [netdata/netdata#13667]
### Bug description

Hey guys, copy pate from netdata dash to install on k8s cluster and getting error below:

Error: INSTALLATION FAILED: unable to build kubernetes objects from release manifest: resource mapping not found for name: "netdata-psp" namespace: "" from "": no matches for kind "PodSecurityPolicy" in version "policy/v1beta1"
ensure CRDs are installed first

k8s version v1.25.0

### Expected behavior

netdata installed!

### Steps to reproduce

1.
2.
3.
...

### Installation method

helmchart (kubernetes)

### System info

```shell
uname -a; grep -HvE "^#|URL" /etc/*release
Linux worker01 5.15.0-47-generic #51-Ubuntu SMP Thu Aug 11 07:51:15 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
/etc/lsb-release:DISTRIB_ID=Ubuntu
/etc/lsb-release:DISTRIB_RELEASE=22.04
/etc/lsb-release:DISTRIB_CODENAME=jammy
/etc/lsb-release:DISTRIB_DESCRIPTION="Ubuntu 22.04.1 LTS"
/etc/os-release:PRETTY_NAME="Ubuntu 22.04.1 LTS"
/etc/os-release:NAME="Ubuntu"
/etc/os-release:VERSION_ID="22.04"
/etc/os-release:VERSION="22.04.1 LTS (Jammy Jellyfish)"
/etc/os-release:VERSION_CODENAME=jammy
/etc/os-release:ID=ubuntu
/etc/os-release:ID_LIKE=debian
/etc/os-release:UBUNTU_CODENAME=jammy
```

### Netdata build info

```shell
N/A
```

**Comment by BillKalaitzis** [comment]
Hey @marliyev,

This seems like an API incompatibility issue.
Since Kubernetes v1.25, the `PodSecurityPolicy` object has been [deprecated](https://kubernetes.io/docs/reference/using-api/deprecation-guide/).
We'll need to do some work to migrate away from it on our helmchart and we'll get back to you!

cc @netdata/cloud-sre @ilyam8

**Comment by marliyev** [comment]
cool, waiting for update, thnx

**Comment by ilyam8** [comment]
Thanks for reporting, @marliyev. Fixed in netdata/helmchart#308.
